### ISSUE TITLE
Only allow users with credit to post on the main forums

### ISSUE THREAD

**Issue by ChipWolf** [minecrafthome/minecrafthome#20]
This should protect those boards from spammers.

**Comment by MrSpike63** [comment]
https://boinc.berkeley.edu/trac/wiki/ProtectionFromSpam
```
forum.post_min_expavg_credit
forum.post_min_total_credit
```
These need to be manually set in the database. You could just set the minimum credit to 1.

**Comment by ChipWolf** [comment]
Yeah, we shouldn't manually set anything in the database, it needs to be in this repo somewhere
